### DISCUSSION TITLE
@Oh my goodness!!!!!!! AdGuard: Immediate Rectification Demanded for Critical Service Disruption

### DISCUSSION THREAD

**Discussion by jameschung0225** [List-KR/community#24] [General]
Oh my goodness!!!!!!!
AdGuard service is causing a catastrophic disruption to a vital banking application in Korea, making it completely non-functional due to the blocking of the DNS resolution for mbp.hanabank.com. This level of service failure is absolutely intolerable. 

Immediate and unambiguous action is demanded to rectify this issue. This is not a matter of inconvenience, but of potential financial damage affecting countless individuals who rely on this service daily. Any additional delay in resolving

**Comment by piquark6046** [comment]
Duplicate of List-KR/community#17
